EU4toV2/Source/Mappers/FactoryInventions/FactoryInventionMapper.h

```diff
@@ -1,25 +1,23 @@
 #ifndef FACTORY_INVENTION_MAPPER_H
 #define FACTORY_INVENTION_MAPPER_H
-
 #include "Parser.h"
-#include <map>
 
 namespace mappers
 {
 
-	class FactoryInventionMapper : commonItems::parser
-	{
-	public:
-		FactoryInventionMapper();
-		explicit FactoryInventionMapper(std::istream& theStream);
-		
-		[[nodiscard]] const auto& getFactoryInventionMap() const { return factoryInventionMap; }
+class FactoryInventionMapper: commonItems::parser
+{
+  public:
+	FactoryInventionMapper();
+	explicit FactoryInventionMapper(std::istream& theStream);
+
+	[[nodiscard]] const auto& getFactoryInventionMap() const { return factoryInventionMap; }
```

Comment:
Investigate whether it makes sense to return the map, or provide a lookup function instead.